pkgs/tools/text/dcs/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, python3
+, perl
+, zopfli
+}:
+buildGoModule {
+  pname = "dcs";
+  version = "unstable-2021-04-07";
+
+  src = fetchFromGitHub {
+    owner = "Debian";
+    repo = "dcs";
+    rev = "da46accc4d55e9bfde1a6852ac5a9e730fcbbb2c";
+    sha256 = "N+6BXlKn1YTlh0ZdPNWa0nuJNcQtlUIc9TocM8cbzQk=";
+  };
+
+  vendorSha256 = "l2mziuisx0HzuP88rS5M+Wha6lu8P036wJYZlmzjWfs=";
+
+  # Depends on dcs binaries
+  doCheck = false;
+
+  nativeBuildInputs = [
+    python3.pkgs.slimit
```

Comment:
```suggestion
    python3Packages.slimit
```

Reply:
Any reason in particular? They're the same, AFAICS.

Reply:
I think python3Packages might be easier to overwrite but the difference is minimal.

Reply:
That's reasonable enough.